gatsby-theme-portfolio-minimal/src/sections/About/index.tsx

```diff
@@ -15,15 +15,20 @@ export function AboutSection(props: PageSection): React.ReactElement {
             <Section anchor={props.sectionId} heading={props.heading}>
                 <div className={classes.About}>
                     <div className={classes.Description} dangerouslySetInnerHTML={{ __html: data.html }} />
-                    <Animation type="fadeLeft" delay={200}>
-                        <div className={classes.ImageWrapper}>
-                            <GatsbyImage
-                                image={data.frontmatter.imageSrc.childImageSharp.gatsbyImageData}
-                                className={classes.Image}
-                                alt={data.frontmatter.imageAlt || `About Image`}
-                            />
-                        </div>
-                    </Animation>
+                        {data.frontmatter.imageSrc ?
```

Comment:
literally my first time writing react so if there's a more idiomatic way to do a conditional, please lmk.